src/sentry/incidents/tasks.py

```diff
@@ -112,11 +123,13 @@ def build_activity_context(activity, user, activity_user=None):
 
 
 @register_subscriber(SUBSCRIPTION_METRICS_LOGGER)
-def handle_subscription_metrics_logger(subscription_update, subscription):
+def handle_subscription_metrics_logger(
+    subscription_update: SubscriptionUpdate, subscription: QuerySubscription
+) -> None:
     """
     Logs results from a `QuerySubscription`.
     :param subscription_update: dict formatted according to schemas in
-    sentry_kafka_schemas.schema_types.events_subscription_results_v1.SubscriptionResults
+    sentry_kafka_schemas.schema_types.events_subscription_results_v1.SubscriptionResult
     :param subscription: The `QuerySubscription` that this update is for
     """
```

Comment:
Don't think you need these if there are types. It's a `SubscriptionUpdate` now since it's been transformed